eclair-core/src/main/resources/reference.conf

```diff
@@ -32,7 +32,7 @@ eclair {
 
   min-feerate = 3 // minimum feerate in satoshis per byte
   smooth-feerate-window = 6 // 1 = no smoothing
-  feerate-provider-timeout = 5 seconds // max time we'll wait for answers from a fee provider before we fallback to the next one
+  feerate-provider-timeout = 15 seconds // max time we'll wait for answers from a fee provider before we fallback to the next one
```

Comment:
I think we should leave it to 5 seconds, it seems reasonable for server environments. 

Of course for mobile apps, that can be increased to 15 seconds but that change should be done in the application's own configuration file.

Reply:
If you prefer, I checked and currently eclair-mobile / phoenix don't increase these values. I can open a separate PR on eclair-mobile / phoenix to do the increase, how does that sound?